Where2Play/Pages/Shared/_Layout.cshtml

```diff
@@ -13,7 +13,7 @@
     <header>
         <nav class="navbar navbar-expand-lg fixed-top">
             <div class="container-fluid">
-                <a class="navbar-brand me-auto" href="index">Where2Play</a>
+               <a class="navbar-brand me-auto" asp-page="/Index">Where2Play</a> 
```

Comment:
I could go either way on href vs asp-page... I think you'll get the same results either way.